WordPress/Classes/ViewRelated/Domains/Register/RegisterDomainDetails/ViewModel/RegisterDomainDetailsViewModel.swift

```diff
@@ -464,8 +464,17 @@ class RegisterDomainDetailsViewModel {
     private func formattedPhoneNumber() -> String {
         let section = sections[SectionIndex.phone.rawValue]
         let countryCode = section.rows[CellIndex.PhoneNumber.countryCode.rawValue].editableRow?.value ?? ""
+
+        let strippedCountryCode = countryCode
+            .replacingOccurrences(of: Constant.phoneNumberCountryCodePrefix, with: "")
+            .replacingOccurrences(of: "00", with: "")
```

Comment:
we can add a check here to replace the 00 only if it is the prefix, it is a very very rare case but I see a country code as 500 here https://www.internationalcitizens.com/international-calling-codes/